packages/playground/blueprints/src/lib/steps/apply-wordpress-patches/index.ts

```diff
@@ -199,6 +201,12 @@ class WordPressPatcher {
 			`${this.wordpressPath}/wp-content/mu-plugins/3-links-targeting-top-frame-should-target-playground-iframe.php`,
 			linksTargetingTopFrameShouldTargetPlaygroundIframe
 		);
+
+		// Activate URL rewriting.
+		await this.php.writeFile(
+			`${this.wordpressPath}/wp-content/mu-plugins/got-url-rewrite.php`,
```

Comment:
Let's rename the file to `4-enable-url-rewrite.php` so that the name communicates the purpose